include/nbl/builtin/hlsl/bxdf/transmission/delta_distribution.hlsl

```diff
@@ -59,7 +37,12 @@ struct SDeltaDistribution
     sample_type generate(NBL_CONST_REF_ARG(anisotropic_interaction_type) interaction, const vector2_type u)
     {
         ray_dir_info_type L = interaction.getV().transmit();
-        return sample_type::createFromTangentSpace(L, interaction.getFromTangentSpace());
+        sample_type s = sample_type::create(L, interaction.getN());
+        s.TdotL = hlsl::dot(L.getDirection(),interaction.getT());
+        s.BdotL = hlsl::dot(L.getDirection(),interaction.getB());
```

Comment:
its an anisotropic interaction, you already have those dot products.

during tranmission you stay in the same plane spanned by V and N as reflection, so again the dot products of L with TB are negatives of V's